docs/BR.md

```diff
@@ -1008,6 +1042,83 @@ CAs are permitted to treat a record lookup failure as permission to issue if:
+| Agreed-Upon Change to Website - ACME (defined in 3.2.2.4.19) | This method involves observing changes to website content.<br><br>“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+| TLS Using ALPN (defined in 3.2.2.4.20) | This method involves establishing a TLS connection.<br><br>“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+| Agreed-Upon Change to Website (defined in 3.2.2.5.1) | This method involves observing changes to website content.<br><br>“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+| Reverse Address Lookup (defined in 3.2.2.5.3) | This method involves a DNS lookup to obtain information about which domain is associated with an IP address.<br><br>“Corroborate” means the domain name retrieved via the reverse DNS lookup MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued.
+| ACME “http-01” method for IP Addresses (defined in 3.2.2.5.6) | This method involves observing changes to website content<br><br>“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+| ACME “tls-alpn-01” method for IP Addresses (defined in 3.2.2.5.7) | This method involves establishing a TLS connection.<br><br>
+“Corroborate” means the corresponding challenge information (e.g., Random Value or Request Token) MUST be observed by the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table, otherwise the certificate MUST NOT be issued. |
+
+
+Table: Defining Corroborating CAA Evidence
+
+| CAA Method | Description of Corroborating Evidence |
+|---|---|
+| Any method | “Corroborate” means the required number of Network Perspectives defined in the "# of Network Perspectives and Quorum Requirements" table MUST have CAA lookup results that permit issuance for the domain in question. |
+
+Computing systems performing Multi-Perspective Domain Validation:
```

Comment:
Isn't "Computing systems performing MPDV" just a synonym for "Network Perspectives"?

Reply:
^ Intent of the language is as described by @dkluge (define minimum security requirements for the hosts running the perspectives).

Reply:
[Closing stale thread, clarification shared.]

@aarongable, feel free to re-open if you disagree.